docs/api/form.md

```diff
@@ -4,12 +4,12 @@ title: <Form />
 custom_edit_url: https://github.com/jaredpalmer/formik/edit/master/docs/api/form.md
 ---
 
-Form is a small wrapper around an HTML `<form>` element that automatically hooks into Formik's `handleSubmit`. All other props are passed directly through to the DOM node.
+Form is a small wrapper around an HTML `<form>` element that automatically hooks into Formik's `handleReset` and `handleSubmit`. All other props are passed directly through to the DOM node.
```

Comment:
I would reverse the order here. handleSubmit should come first and reset second?

Reply:
Yep that sounds good, I just went with alphabetical but agree that submit is more important/common and should go first.

Reply:
Question for you. How does this work with buttons without `type`? Just confirming that submit still takes precedence, yes?

Reply:
It should be the same default behavior, which iirc is browser-dependent. I haven’t tested it super thoroughly, but this should only affect buttons that explicitly have type reset. (No browsers default to reset, only submit or button.)